packages/swap/src/routes/swap.ts

```diff
@@ -248,6 +251,7 @@ router.get(
       ccmDepositReceivedBlockIndex: swap?.ccmDepositReceivedBlockIndex ?? undefined,
       ccmMetadata,
       depositChannelOpenedThroughBackend: swapDepositChannel?.openedThroughBackend,
+      broadcastTxRef: swap?.egress?.broadcast?.transactionRef ?? egressTrackerTxRef,
```

Comment:
we dont abbreviate `transaction` in `depositTransactionHash` and `depositTransactionConfirmations`, so i would not do it here too